SVSModel/Configuration/SoilTestConfig.cs

```diff
@@ -35,31 +41,15 @@ public double BulkDensity
         get
         {
             double soilMF = 1;
-            if (TypeOfTest.ToString()==TestType.QuickTest.ToString())
-                soilMF = Constants.MoistureFactor[TextureOfSoil.ToString()][MoistureOfSample.ToString()];
-            double soilDepthFactor = SampleDepthFactor[DepthOfSample];
-            double result = TestValue / soilMF * BulkDensity * 3 * soilDepthFactor;
-            return new Dictionary<DateTime, double>() { { TestDate, result } };
+            if (TypeOfTest == TestType.QuickTest.ToString())
```

Comment:
could just replace TestType.QuickTest.ToString() with "QuickTest".  The longer winded way is a hangover from when I was using enums

Reply:
I think I am fine with leaving it like this for now.
Using enums is certainly cleaner in the long run, but it will require some refactoring on the webapp backend to accommodate.